### ISSUE TITLE
False positive with comment in `no-invalid-block-param-definition` rule

### ISSUE THREAD

**Issue by bmish** [ember-template-lint/ember-template-lint#1539]
The [no-invalid-block-param-definition](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-invalid-block-param-definition.md) rule should ignore the `as` in comments right?

```hbs
<div {{! This is needed to serve as a container }}>
</div>
```
